CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## WLED changelog
 
+#### Build 2410270
+-   WLED 0.15.0-b7 release
+-   Add visual expand button on hover (#4172)
+-   `/json/live` (JSON live data/peek) only enabled when WebSockets are disabled
+-   Bugfixes: #4179, #4215, #4219, #4224, #4228, #4230
+
```

Comment:
Please add two items for audioreactive:
* AR usermod: memory optimizations - only use RAM when enabled (by @softhack007)
* AR usermod: fix for major performance degradation in FFT